---a/net/minecraft/world/gen/feature/WorldGenDeadBush.java
+++b/net/minecraft/world/gen/feature/WorldGenDeadBush.java
@@ -1,34 +1,32 @@
 package net.minecraft.world.gen.feature;
 
 import java.util.Random;
+
 import net.minecraft.block.Block;
 import net.minecraft.block.material.Material;
 import net.minecraft.init.Blocks;
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 
-public class WorldGenDeadBush extends WorldGenerator
-{
+public class WorldGenDeadBush extends WorldGenerator {
+
     private static final String __OBFID = "CL_00000406";
 
-    public boolean generate(World worldIn, Random p_180709_2_, BlockPos p_180709_3_)
-    {
+    public boolean generate(World worldIn, Random p_180709_2_, BlockPos p_180709_3_) {
         Block block;
 
-        while (((block = worldIn.getBlockState(p_180709_3_).getBlock()).getMaterial() == Material.air || block.getMaterial() == Material.leaves) && p_180709_3_.getY() > 0)
-        {
+        while (((block = worldIn.getBlockState(p_180709_3_).getBlock()).getMaterial() == Material.air || block.getMaterial() == Material.leaves) && p_180709_3_.getY() > 0) {
             p_180709_3_ = p_180709_3_.down();
         }
 
-        for (int i = 0; i < 4; ++i)
-        {
+        for (int i = 0; i < 4; ++i) {
             BlockPos blockpos1 = p_180709_3_.add(p_180709_2_.nextInt(8) - p_180709_2_.nextInt(8), p_180709_2_.nextInt(4) - p_180709_2_.nextInt(4), p_180709_2_.nextInt(8) - p_180709_2_.nextInt(8));
 
-            if (worldIn.isAirBlock(blockpos1) && Blocks.deadbush.canBlockStay(worldIn, blockpos1, Blocks.deadbush.getDefaultState()))
-            {
+            if (worldIn.isAirBlock(blockpos1) && Blocks.deadbush.canBlockStay(worldIn, blockpos1, Blocks.deadbush.getDefaultState())) {
                 worldIn.setBlockState(blockpos1, Blocks.deadbush.getDefaultState(), 2);
             }
         }
 
         return true;
     }
-}
+}
